src/components/SeekHelpCarousel/index.jsx

```diff
@@ -31,6 +32,11 @@ const SeekHelpCarousel = () => {
           </Slider>
         </Col>
       </Row>
+      <Link to="/seekhelp">
+        <h6 className="searchHelpBtn">
```

Comment:
<h6>
            {t("home.buttons. searchHelpBtn")}
            {document.body.dir === "rtl" ? (
              <span> &#8592; </span>
            ) : (
              <span> &#8594; </span>
            )}
          </h6>

Reply:
okay

Reply:
can you make the code like this so the arrow flips between Arabic and English and check if its work

Reply:
looks like this
<img width="1440" alt="Screen Shot 2020-11-22 at 21 08 24" src="https://user-images.githubusercontent.com/67760504/99911662-0497a480-2d07-11eb-90f7-b4b1ecab8867.png">

Reply:
but should be like this I guess, right?
<img width="1440" alt="Screen Shot 2020-11-22 at 21 09 06" src="https://user-images.githubusercontent.com/67760504/99911677-1d07bf00-2d07-11eb-98d6-cec0aae0c400.png">

Reply:
try adding it in a row

Reply:
is second or first option right? i mean arrows